pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
```

Comment:
Instead of "Output is formatted", use "Format output" (we should probably mention the recommendation to use active rather than passive voide in the contributing guidelines)